### PR TITLE
Html Structure for Todo-List Project

### PR DESCRIPTION
**Pull request by Qoosim**
I have added Html structure for a good UI appearance.
I created an array of objects with three properties - description, completed, and index that would get displayed on the browser when it is loaded.
I also created a function that will iterate over the array of objects and display the description properties with the checkboxes at the left side that could be used to indicate whether a task is completed or not when it is checked or unchecked.
I also added icons that can be used later to perform certain operations on a particular task.

### CODE REVIEW

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @Qoosim  :raised_hands: 
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for this project.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Very descriptive commit messages and pull request summary
:heavy_check_mark: Webpack setup :100: 

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Optional comments are not required to mark this project approved
- [ ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .hintrc, .eslintrc.json etc).


**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,18 +1,5 @@
...
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-clone the project and you can start your own project on it.
-
 ## Authors
```
[the remark this answers is not in the document]
Comment (henatan99) [reply README.md]: This issue is well addressed. Nice work!

File: src/index.js [hunk 2]
```diff
@@ -1,13 +1,47 @@
-import _ from 'lodash'; 
+import _ from 'lodash';// eslint-disable-line no-unused-vars
```
[the remark this answers is not in the document]
Comment (henatan99) [reply src/index.js]: This issue is well addressed. Great job! :+1:

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (henatan99) [reply README.md]: - [ ] This issue is not addressed completely. The project title and description are not added as per the change requested by the previous reviewer. Kindly make this change as it is a mandatory general requirement. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Qoosim , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [ ] You did a great job in general, but it looks like you forgot to address one change requested by the previous reviewer. The project title and description are not added in your readme as per the change requested by the previous reviewer. Kindly make this change as it is a mandatory general requirement. You can read the general requirement for a professional repo [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
image

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by carlos-ssh** [comment]
Hi @Qoosim , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that Your branch contains conflicts that must be resolved.. Kindly work on it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**